react/Media/styles.styl

```diff
@@ -15,3 +15,22 @@
 .img
     line-height 0
     flex 0 0 auto
+
+:global
```

Comment:
Because I find the way utility classes are declared hard to do :) I put it like that to see if `.u-media-grow` would stick and it pleased other people.

Reply:
I fixed this but put the commit in another PR #1213